sdk/data/azcosmos/cosmos_change_feed_composite_continuation_token_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcosmos
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+)
+
+func TestNewCompositeContinuationToken(t *testing.T) {
+	resourceID := "testResource"
+	continuationToken := azcore.ETag("14")
+	changeFeedRangeInstance := newChangeFeedRange(
+		"",
+		"FF",
+		&ChangeFeedRangeOptions{
+			ContinuationToken: &continuationToken,
+		},
+	)
+
+	compositeContinuationToken := newCompositeContinuationToken(resourceID, []changeFeedRange{changeFeedRangeInstance})
+	t.Logf("ResourceID: %s\nContinuation: %+v", compositeContinuationToken.ResourceID, compositeContinuationToken.Continuation)
+
+	data, err := json.Marshal(compositeContinuationToken)
+	if err != nil {
+		t.Fatalf("Failed to marshal composite token: %v", err)
+	}
+
+	expectedJSON := `{"resourceId":"testResource","continuation":[{"minInclusive":"","maxExclusive":"FF","continuationToken":"14"}]}`
```

Comment:
From what I can tell, both the [Java ](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/implementation/feedranges/FeedRangeContinuationTest.java) and [Python](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/cosmos/azure-cosmos/azure/cosmos/_change_feed/feed_range_composite_continuation_token.py#L35) SDKs use the same JSON format (though the casing of property names is a little different). We should probably use the same format here, even though I don't think there's much reason to share continuation tokens between SDKs.